### DISCUSSION TITLE
Placeholder on Bank Tag Tabs Issues

### DISCUSSION THREAD

**Discussion by CleetZ** [runelite/runelite#16726] [Q&A]
Hello. When You withdraw a fire cape or defender from the tag tab it doesnt have a placeholder with a (0) remaining in the tag tab. Theres probly other items where this bug occurs also but those are the ones I know for sure. Like I'll withdraw my granite maul and it holds the place in the tag tab, but not for the fire cape or defender. Not a huge deal but definitely a little quality of life feature because I get confused sometimes, plus it would just look better if it had a placeholder when You withdraw it. Thanks for Your time

**Comment by geheur** [comment]
have you tried adding the item to the tab by holding shift while dragging it onto the tab? This adds different item ids of similar items.

**Comment by raiyni** [comment]
You need set them as placeholders. If you need client support join our discord.

**Comment by CleetZ** [comment]
Nah I had the fire cape and defender set as place holders, but when its in a bank tag tab, the placeholder doesnt stay, they still dissapear

Sent with [Proton Mail](https://proton.me/) secure email.

------- Original Message -------
